indico/modules/events/export.yaml

```diff
@@ -477,6 +485,7 @@ import:
     ContributionPrincipal.user_id: skip
     AttachmentPrincipal.user_id: skip
     AttachmentFolderPrincipal.user_id: skip
+    MenuEntryPrincipal.user_id: skip
```

Comment:
When the user chooses to not import missing users during the event import, then this skips the MenuEntryPrincipal object altogether (it's pointless to import a user acl entry if that user does not exist on the target indico instance)